stable/mlrun/values.yaml

```diff
@@ -138,8 +138,8 @@ serviceAccounts:
 v3io: {}
 
 httpDB:
-  dbType: "filerundb"
+  dbType: "sqldb"
   dirPath: "/mlrun/db"
-  dsn: "sqlite:///mlrun.sqlite3?check_same_thread=false"
+  dsn: "sqlite:////mlrun/db/mlrun.db?check_same_thread=false"
```

Comment:
ass-plain the `////` ?

Reply:
https://docs.sqlalchemy.org/en/13/dialects/sqlite.html:
>An absolute path, which is denoted by starting with a slash, means you need four slashes:

Reply:
so far we were using filedb so this dsn had no meaning, when we started using sqldb we actually did this in MLRun - https://github.com/mlrun/mlrun/blob/development/mlrun/config.py#L220 - (practically although the value is `sqlite:///mlrun.sqlite3?check_same_thread=false` this value will be used - `sqlite:////mlrun/db/mlrun.db?check_same_thread=false`
so what I'm doing here is just to align the helm chart so that the value we see is the value really running

Reply:
understood